### PR TITLE
Doc/minor improvement

### PR DESCRIPTION
**Pull request by AMontagu**
## Description

Added minor improvement as specified in https://github.com/grafana/faro-web-sdk/issues/131#issuecomment-1470427538

Also integrate https://github.com/grafana/faro-web-sdk/pull/226 but keep separated if some change in this one are not wanted. If wanted 226 can be close and change made here or 226 can be merge and this rebased after it.

### CODE REVIEW

File: docs/sources/tutorials/quick-start-browser.md [hunk 1]
```diff
@@ -191,10 +192,38 @@ context.with(trace.setSpan(context.active(), span), () => {
 });
 ```
 
+### With OpenTelemetry tracing on backend on different domain than frontend
+
+By default opentelemetry's XMLHttpRequestInstrumentation and FetchInstrumentation
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:197]:
```suggestion
By default, OpenTelemetry's XMLHttpRequestInstrumentation and FetchInstrumentation inject the traceparent header only into requests sent to the same origin as the current window. To this functionality to other origins, you can enable it by setting up propagateTraceHeaderCorsUrls as shown below.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 2]
```diff
@@ -191,10 +192,38 @@ context.with(trace.setSpan(context.active(), span), () => {
...
+only inject traceparent header to request send to the same domain than the current windows.
+To enable it to others backend please set up propagateTraceHeaderCorsUrls as show below.
+
+```ts
+import { getWebInstrumentations, initializeFaro } from '@grafana/faro-web-sdk';
+import { TracingInstrumentation } from '@grafana/faro-web-tracing';
+
+const instrumentationOptions = {
+  propagateTraceHeaderCorsUrls: [new RegExp("BACKEND_SERVER(S)_REGEX")] // This is a list of spécifique URI or regex
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:206]: // This is a list of specific URIs or regular exprressions
```suggestion
  propagateTraceHeaderCorsUrls: [new RegExp("BACKEND_SERVER(S)_REGEX")] // This is a list of spécifique URI or regex 
```

**Comment by AMontagu** [comment]
@codecapitano Reviews done

File: docs/sources/tutorials/quick-start-browser.md [hunk 3]
```diff
@@ -191,10 +192,38 @@ context.with(trace.setSpan(context.active(), span), () => {
...
+
 ### With custom OpenTelemetry tracing configuration
 
-The following example configure OTel manually and uses `FaroTraceExporter` and call `faro.api.initOTEL` with OTel trace
-and context APIs.
+The following example configure OTel manually and uses `FaroTraceExporter` and
+call `faro.api.initOTEL` with OTel traceand context APIs.
+If you just want to change wich instrumentations is used, please use the
+[instrumentations option](https://github.com/grafana/faro-web-sdk/blob/main/packages/web-tracing/src/types.ts)
+of TracingInstrumentation class.
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:227]:
```suggestion
The following example, demonstrates how to configure OpenTelemetry manually using the `FaroTraceExporter`. To achieve this, we call the `faro.api.initOTEL` function with the `OTELTraceAPI` and `OTELContextAPI` as parameters.

If you want to change or modify the instrumentations used, you can easily achieve this by utilizing the [instrumentations option](https://github.com/grafana/faro-web-sdk/blob/main/packages/web-tracing/src/types.ts) within the `TracingInstrumentation` class.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 4]
```diff
@@ -244,6 +273,9 @@ provider.register({
 
 const ignoreUrls = [COLLECTOR_URL];
 
+// please note that this instrumentation come from open telemetry and can't be used directly
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:276]: Please be aware that this instrumentation originates from OpenTelemetry and cannot be used directly in the initializeFaro instrumentations options. If you wish to configure these instrumentations using the initializeFaro function, please utilize the instrumentations options within the TracingInstrumentation class.

File: docs/sources/tutorials/quick-start-browser.md [hunk 5]
```diff
@@ -322,6 +354,31 @@ faro.pause();
 faro.unpause();
 ```
 
+## How to activate debugging
+
+Enable faro internal logger:
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:359]:
```suggestion
To enable the Faro internal logger, follow these steps:
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 6]
```diff
@@ -322,6 +354,31 @@ faro.pause();
...
+import { initializeFaro, InternalLoggerLevel } from '@grafana/faro-web-sdk';
+
+const faro = initializeFaro({
+  url: 'https://collector-host:12345/collect',
+  apiKey: 'secret',
+  app: {
+    name: 'frontend',
+    version: '1.0.0',
+  },
+  internalLoggerLevel: InternalLoggerLevel.VERBOSE, // Values possible: OFF, ERROR, WARN, INFO, VERBOSE
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:371]:
```suggestion
  internalLoggerLevel: InternalLoggerLevel.VERBOSE, // Possible values are: OFF, ERROR, WARN, INFO, VERBOSE
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 7]
```diff
@@ -322,6 +354,31 @@ faro.pause();
...
+});
+```
+
+Enable opentelemetry debug logger:
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:375]:
```suggestion
To enable the OpenTelemetry debug logger, follow these steps:
```

**Review by codecapitano — CHANGES_REQUESTED** [review]
Hi @AMontagu this is a great enhancement of the docs 🙏 
Just some minor changes to do.

**Comment by codecapitano** [comment]
@AMontagu the built is failing due to some Linter errors (Line 197). I think most of them are related to line length which must not exceed 120 characters.

**Comment by AMontagu** [comment]
@codecapitano Change requested done, commit squased and CI passing.

File: docs/sources/tutorials/quick-start-browser.md [hunk 8]
```diff
@@ -191,10 +191,43 @@ context.with(trace.setSpan(context.active(), span), () => {
 });
 ```
 
+### With OpenTelemetry tracing on backend on different domain than frontend
+
+By default, OpenTelemetry's XMLHttpRequestInstrumentation and FetchInstrumentation
+inject the traceparent header only into requests sent to the same origin as the current window.
+To this functionality to other origins, you can enable it by setting up propagateTraceHeaderCorsUrls as shown below.
+only inject traceparent header to request send to the same domain than the current windows.
+To enable it to others backend please set up propagateTraceHeaderCorsUrls as show below.
```
Comment (codecapitano) [inline docs/sources/tutorials/quick-start-browser.md:200]:
```suggestion
By default, OpenTelemetry's XMLHttpRequestInstrumentation and FetchInstrumentation
inject the traceparent header only into requests send to the same origin as the current window.
To enable this functionality for other origins, you can enable it by setting up propagateTraceHeaderCorsUrls as shown below.
```

**Review by codecapitano — CHANGES_REQUESTED** [review]
Hi @AMontagu there's only a small thing left. I think this happened by auto applying the suggestion.

**Review by codecapitano — APPROVED** [review]
LGTM 🦖 Thank you so much for your contribution and patience  🙏

**Pull request merged**